_data/tools.yml

```diff
@@ -256,6 +256,18 @@
   v3: true
   v3_1: true
 
+- name: OpenAPI-UI
+  category: documentation
+  description:
+    a simpler and more beautiful openapi UI document than swagger UI, 
+    quickly generate mock params and call api request.
+  link: https://docs.openapi-ui.com
```

Comment:
Great feature, just a suggestion: a default OpenAPI spec in 2.0, 3.0, 3.1 standards would make it easier to see what your tool does.

Reply:
I was just offering a suggestion, purely my own, that having default specs in various OpenAPI versions that users could just click on would let your users see instantly how your tool works. Just food for thought, and nothing more.

Reply:
Maybe I should automatically fill in an openapi-spec to support users to quickly pre-render?

Reply:
I was thinking just having PetStore-default OpenAPI specs in various versions available on your site. Then users could click on it (without the bother of having their own, uploading etc) and see what your tool looks like instantly.

This is completely off-topic and just a personal suggestion. It has no relation to this review, please disregard or factor as you wish :).

Reply:
I understand what you mean. There is actually a link to the openapi-spec in the form-label. Maybe I should fill it in by default